ovirt_imageio/client/_utils.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: Red Hat, Inc.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+"""
+Utility module holding helper functions.
+"""
+
+import shutil
+import tempfile
+
+from contextlib import contextmanager
+
+
+@contextmanager
+def tmp_dir(prefix):
```

Comment:
We already have _internal/util.py - why not add it there? This code is used both by the client (client._api) and server side (docker/entrypoint.py) so it should bn in the common package.